lib/builtIn/Macro.js

```diff
@@ -27,6 +27,9 @@ class Macro extends DefaultConfig {
      * @returns {boolean|*} FALSE if the given element should be filtered out.
      */
     preFilterScenario(scenario, parent, i) {
+        if (!scenario.tags) {
```

Comment:
Could you please create a simple test case for this branch?

Reply:
forgot about that.. sure!